crates/store/re_types/definitions/rerun/blueprint/components/timeline.fbs

```diff
@@ -0,0 +1,14 @@
+
+namespace rerun.blueprint.components;
+
+/// A timeline, identified by its name.
+table Timeline (
+  "attr.arrow.transparent",
+  "attr.python.aliases": "str",
+  "attr.python.array_aliases": "str, Sequence[str]",
+  "attr.rust.derive": "Default, PartialEq, Eq, PartialOrd, Ord",
```

Comment:
This `Default` implementation is bad (yield `""` empty timeline name), but is required by reflection.

Reply:
You can remove it from here and add a `impl Default for Timeline` in `timeline_ext.rs` instead and return a better default (e.g. `log_time`)